Extension/tools/OptionsSchema.json

```diff
@@ -561,7 +578,7 @@
             "%c_cpp.debuggers.cppvsdbg.console.newExternalWindow.description%"
           ],
           "description": "%c_cpp.debuggers.cppvsdbg.console.description%",
-          "default": "internalConsole"
+          "default": "externalTerminal"
```

Comment:
This is going to mismatch when the user does not provide `console`. vsdbg will default to `internalConsole`.

Reply:
@WardenGnaw Why is package.json "default" set to "externalTerminal" then? Isn't that a mismatch?

Reply:
When I run generateOptionsSchema this gets updated.